modules/plugins/theme/supported-themes.nix

```diff
@@ -4,7 +4,15 @@
 }: let
   inherit (lib.strings) optionalString;
   inherit (lib.trivial) boolToString warnIf;
+  inherit (lib.nvim.lua) toLuaObject;
+  cfg = config.vim.theme;
 in {
+  base16 = {
+    setup = {base16-colors, ...}: ''
+      -- Base-16 theme
+      require('base16-colorscheme').setup(${toLuaObject cfg.base16-colors})
```

Comment:
Why even take the base16-colors argument if we just get them from cfg anyway? It's one or the other, and I'd go with the argument for consistency's sake.